### PR TITLE
Add tutorials before initial release

### PR DESCRIPTION
**Pull request by yannmclatchie**
## Description

This PR will add initial tutorials required for an initial package release.

## Checklist

- [ ] Tests covering the new functionality have been added
- [ ] Documentation has been updated OR the change is too minor to be documented
- [ ] Changes are listed in the `CHANGELOG.md` OR changes are insignificant

### CODE REVIEW

File: docs/notebooks/tutorials/intro.ipynb [hunk 1]
```diff
@@ -0,0 +1,556 @@
+{
```
Comment (aloctavodia) [inline docs/notebooks/tutorials/intro.ipynb:1]: <p>split imports from the data generation</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (aloctavodia) [inline docs/notebooks/tutorials/intro.ipynb:1]: <p>is to find a subset of candidate</p><p> </p><p>Could you clarify point 2? "a model more robust to changes in the data-generating distribution"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (aloctavodia) [inline docs/notebooks/tutorials/intro.ipynb:1]: <p>Justify manual prior</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (aloctavodia) [inline docs/notebooks/tutorials/intro.ipynb:1]: <p>Add all imports to the same cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (aloctavodia) [inline docs/notebooks/tutorials/intro.ipynb:1]: <p>change "In essence, the "projection" stage of our model selection procued is achieved by fitting to the fit of the reference model. "</p><p> </p><p>to something like </p><p> </p><p>In essence, the "projection" stage is done by fitting models to samples from the posterior predictive distribution of the reference model.</p><p> </p><p>So L1 for up to 40 covariates and forward for more than 40?</p><p> </p><p>maybe remove this "(possibly coarsened by clustering or thinning the posterior draws) "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (aloctavodia) [inline docs/notebooks/tutorials/intro.ipynb:1]: <p>"log" scale is the default</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (yannmclatchie) [reply docs/notebooks/tutorials/intro.ipynb:1]: <p>I think I will simply remove point 2 since this will likely result in misunderstanding.</p><p> </p>

Comment (aloctavodia) [reply docs/notebooks/tutorials/intro.ipynb:1]: OK, if possible I still would like to know what do you mean, It sounds interesting, haha

File: docs/notebooks/tutorials/intro.ipynb [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
```
Comment (aloctavodia) [inline docs/notebooks/tutorials/intro.ipynb:1]: <p>what this means "which 5 are informative using <code>sklearn</code>"? Did you use some method from sklearn that gave you that result? Can we should that we should expect only 5 covariates to be informative? Or use another example where we sample x[0:9] from some distribution and then we expliticitelly use X[0:5] to build Y?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (aloctavodia) [inline docs/notebooks/tutorials/intro.ipynb:1]: <p>we need to set a random.seed</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by aloctavodia** [comment]
what else do we need to merge this?

**Pull request merged**